### PR TITLE
Add WiFi basic test

### PR DESCRIPTION
**Pull request by laura-nao**
Add support for a basic WiFi test and run it on an initial set of target Chromebooks, based on the WiFi chip:
- Intel(R) Wi-Fi 6 AX201 160MHz - `asus-cx9400-volteer`
- Intel(R) Wi-Fi 6E AX211 160MHz -  	`acer-cbv514-1h-34uz-brya`
- Intel(R) Wi-Fi 6 AX200 160MHz - `lenovo-TPad-C13-Yoga-zork`
- Intel(R) Dual Band Wireless AC 7265 - `hp-x360-14-G1-sona`
- Intel(R) Wireless-AC 9560 160MHz - `dell-latitude-5400-8665U-sarien`
- Marvell SD8897-B0 - `mt8173-elm-hana`
- RTL8852AE - `acer-cp514-3wh-r0qs-guybrush`
- RTL8822CE - `asus-CM1400CXA-dalboz`
- QCA6174 - `acer-R721T-grunt` , `mt8183-kukui-jacuzzi-juniper-sku16`
- mt7921e - `mt8195-cherry-tomato-r2`

### CODE REVIEW

**Comment by laura-nao** [comment]
Results from reference manual LAVA jobs:
- asus-cx9400-volteer: https://lava.collabora.dev/scheduler/job/12499333
- acer-cbv514-1h-34uz-brya: https://lava.collabora.dev/scheduler/job/12498912
- lenovo-TPad-C13-Yoga-zork: https://lava.collabora.dev/scheduler/job/12499692
- hp-x360-14-G1-sona: https://lava.collabora.dev/scheduler/job/12499696
- dell-latitude-5400-8665U-sarien: https://lava.collabora.dev/scheduler/job/12500108
- mt8173-elm-hana: https://lava.collabora.dev/scheduler/job/12506486
- acer-cp514-3wh-r0qs-guybrush: https://lava.collabora.dev/scheduler/job/12498975
- asus-CM1400CXA-dalboz: https://lava.collabora.dev/scheduler/job/12499655
- acer-R721T-grunt: https://lava.collabora.dev/scheduler/job/12499655
- mt8183-kukui-jacuzzi-juniper-sku16: https://lava.collabora.dev/scheduler/job/12506692
- mt8195-cherry-tomato-r2: https://lava.collabora.dev/scheduler/job/12491913

File: config/core/rootfs-images.yaml [hunk 1]
```diff
@@ -159,6 +159,14 @@ file_systems:
     ramdisk: bullseye-vdso/20230623.0/{arch}/rootfs.cpio.gz
     root_type: ramdisk
     type: debian
+  debian_bullseye-wifi_nfs:
+    boot_protocol: tftp
+    nfs: bullseye-wifi/20240109.0/{arch}/full.rootfs.tar.xz
```
Comment (laura-nao) [inline config/core/rootfs-images.yaml:164]: One note: the name of the image should be updated once a bullseye-wifi rootfs image has been built and is available in the KernelCI storage.
I see some staging jobs being run in LAVA (https://lava.collabora.dev/scheduler/alljobs?length=25&search=wifi#table), but they're all failing as such image does not exist yet.

Comment (pawiecz) [reply config/core/rootfs-images.yaml:164]: Looks like bullseye-wifi rootfs image is available from [2024.01.29](http://storage.kernelci.org/images/rootfs/debian/bullseye-wifi/20240129.0/)

File: config/core/test-configs.yaml [hunk 2]
```diff
@@ -767,6 +767,12 @@ test_plans:
     params:
       job_timeout: '20'
 
+  wifi-basic: &wifi-basic
```
Comment (pawiecz) [inline config/core/test-configs.yaml:770]: To other reviewers: although this anchor is not used anywhere yet, it's a placeholder for more complex wifi test, see e.g. `v4l2-decoder-conformance` in L698

**Review by pawiecz — APPROVED** [review]

**Comment by nuclearcat** [comment]
Merging, as we will have soon major rework to bookworm and we can follow up

**Pull request merged**